engine/lib/configuration.php

```diff
@@ -54,7 +54,8 @@ function elgg_get_data_path() {
  * @return string
  */
 function elgg_get_cache_path() {
-	return _elgg_config()->cacheroot;
+	$path = _elgg_config()->cacheroot ? : elgg_get_data_path() . 'caches/';
```

Comment:
isn't this handled by https://github.com/Elgg/Elgg/pull/11948/files#diff-153aabf87ced6eb965431e48cc8b99ebR749

Reply:
There are use cases when this could be called without the boot, e.g. tests